src/_lib/sass/src/_type.scss

```diff
@@ -9,43 +9,42 @@
 // Headings
```

Comment:
There's a .page-header selector in here that we may have been using incorrectly. The style rules read like it should be applied to a heading tag but it seems like we're often applying this class to divs. Let's discuss what we should do about this